bundles/prepend_extension.rst

```diff
@@ -186,6 +181,12 @@ method::
 
     The ``prependExtension()`` method, like ``prepend()``, is called only at compile time.
 
+.. versionadded:: 7.1
+
+    The :method:`Symfony\\Component\\DependencyInjection\\Loader\\Configurator\\ContainerConfigurator::import`
+    method behavior was modified in Symfony 7.1 to prepend config instead of appending. This behavior change is
+    valid only in the ``prependExtension()`` scope.
```

Comment:
I don't understand this part:

```
This behavior change is valid only in the ...
```

The "is valid only in" part maybe should be "only affects to" ?

Reply:
Updated, it's indeed clearer, thanks!